server/src/bootstrap/index.ts

```diff
@@ -131,6 +136,30 @@ const apiOptions = {
   enquiries: new EnquiryController({ enquiryService, recaptchaService }),
 }
 
+const moduleLogger = createLogger(module)
+
+if (baseConfig.nodeEnv === Environment.Prod) {
+  app.use(
+    connectDatadog({
+      response_code: true,
+      tags: [`service:${datadogConfig.service}`, `env:${datadogConfig.env}`],
```

Comment:
These tags are probably not needed here, since they are set at agent-level

Reply:
You'll probably want to pass in the [unified service tags](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging?tab=kubernetes) (env, service, version) as environment variables, not set them at the agent-level (imagine if you run multiple apps/services on a single EC2, can't assign tags to specific services at the agent level).

Full list of env variables can be found [here](https://docs.datadoghq.com/tracing/setup_overview/setup/nodejs?tab=containers#tagging).

Example:
```
DD_ENV="prod"
DD_SERVICE="example"
DD_VERSION="1.0.1"
DD_LOGS_INJECTION=true
node my_app.js
```

Reply:
Those are actually being set using env vars =) given that we only run one application per EC2 instance (elastic beanstalk after all), this should be okay.

Reply:
I think given our context, either should be okay. It might work cleaner if we set the tags explicitly in the application, but I don't foresee us worrying too much about tag organisation given that for us, a service will constitute an entire product in OGP.